src/screen/home.js

```diff
@@ -99,7 +132,12 @@ function App() {
         </div>
         <main>
           <h2>Enter a city below 👇</h2>
-          <Search setCity={setCity} />
+          <div className="flex search-container">
+            <Search setCity={setCity} />
+            {isLoaded && !notfound?<div className="bookmark-button" onClick={()=>{
+              bookmarkLocation(city);
```

Comment:
The bug I mentioned in bookmarking cities is caused by this line.

- If I have bookmarked three cities Dublin, Sydney and Pune and added a new search for Moscow; the `city` state will update itself to Moscow. 
- Later if I bookmark it, Moscow is bookmarked. Same if I remove it instantly.
- But if I want to remove Pune, which was added **before** Moscow, it gives unexpected output, since the state of the city, which is Moscow, is sent as a parameter.
- The function checks if the city exists or not, and adds it after not finding it in the list.

A possible solution is to keep this button only to add new bookmarks and have a separate button for every card

Reply:
I don't think its a bug its just how we expected it to work currently. If the last searched is MOSCOW you will be able to remove or bookmark only MOSCOW. To do the same for any other city you will have to search for the city separately.
Makes sense?
@joshi-kaushal

Reply:
So if I have five cities, let's say {0,1,2,3,4}, and want to remove city 3, I have to type the city, fetch its weather again, change states, and then remove it. Is that what you are saying?

Sorry, I don't understand😅

Reply:
![image](https://user-images.githubusercontent.com/86974918/181741568-c6bcf9e5-9f94-4736-98c3-158ae3a0dc60.png)
I believe this solves the problem. Made the changes.
@joshi-kaushal